docs/operations/migration_policies.md

```diff
@@ -0,0 +1,161 @@
+    allowAutoConverge: true
+    bandwidthPerMigration: 217Ki
+    completionTimeoutPerGiB: 23
+    allowPostCopy: false
+```
+
+All above fields are optional. When omitted, the behaviour that will apply is the currently defined behaviour in
+KubevirtCR's MigrationConfiguration. This way, KubevirtCR will serve as a configurable set of defaults for both
+VMs that are not bounded to any MigrationPolicy and VMs that are bounded to a MigrationPolicy that does not
+define all the configurations.
+
+#### Matching Policies to VMs
+
+Next in the spec are the selectors that define the group of VM on which to apply the policy. The options to do so
+are the following.
+
+**This policy applies for the VMs in namespaces that have all the required labels:**
+```yaml
+apiVersion: migrations.kubevirt.io/v1alpha1
+kind: MigrationPolicy
+  spec:
+  selectors:
+    namespaceSelector:
+      matchLabels:
+        hpc-workloads: true       # Matches a key and a value 
+        xyz-workloads-type: ""    # Matches the key only, any value applies
+```
+
+**This policy applies for the VMs that have all the required labels:**
+```yaml
+apiVersion: migrations.kubevirt.io/v1alpha1
+kind: MigrationPolicy
+  spec:
+  selectors:
+    virtualMachineInstanceSelector:
+      matchLabels:
+        workload-type: db       # Matches a key and a value 
+        operating-system: ""    # Matches the key only, any value applies
+```
+
+**It is also possible to combine the previous two:**
+```yaml
+apiVersion: migrations.kubevirt.io/v1alpha1
+kind: MigrationPolicy
+  spec:
+  selectors:
+    namespaceSelector:
+      matchLabels:
+        hpc-workloads: true
+        xyz-workloads-type: ""
+    virtualMachineInstanceSelector:
+      matchLabels:
+        workload-type: db
+        operating-system: ""
+```
+
+#### Full Manifest:
+
+```yaml
+apiVersion: migrations.kubevirt.io/v1alpha1
+kind: MigrationPolicy
+metadata:
+  name: my-awesome-policy
+spec:
+  # Migration Configuration
+  allowAutoConverge: true
+  bandwidthPerMigration: 217Ki
+  completionTimeoutPerGiB: 23
+  allowPostCopy: false
+  
+  # Matching to VMs
+  selectors:
+    namespaceSelector:
+      matchLabels:
+        hpc-workloads: true
+        xyz-workloads-type: ""
+    virtualMachineInstanceSelector:
+      matchLabels:
+        workload-type: db
+        operating-system: ""
+```
+## Policies' Precedence
+
+It is possible that a multiple policies apply to the same VMI. In such cases, the precedence is in the
+same order as the bullets above (VMI labels first, then namespace labels). It is not allowed to define
+two policies with the exact same selectors.
+
+If multiple policies apply to the same VMI:
+* The most detailed policy will be applied, that is, the policy with the highest number of matching labels
+* If multiple policies match to a VMI with the same number of matching labels, the policies will be sorted by the
+  lexicographic order of the matching labels keys. The first one in this order will be applied.
```

Comment:
This is not displayed correctly when rendered (see https://deploy-preview-565--kubevirt-user-guide.netlify.app/operations/migration_policies/)
Same with the section below.

Reply:
Thanks for letting me know! And for showing me this cool preview link :D
Fixed